src/population.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+class Population():
+    def __init__(self, solutions, fitness):
+        self.solutions = solutions
+        self.fitness = fitness
+
+    def initial_population(self, population_size, test_size):
+        population = list()
+        for i in range(population_size):
+            test = list()
+            for j in range(test_size):
+                test.append(random.randint(-1000, 1000))
+            population.append(test)
+        return population
+
+    def mutate_1(self, lower, upper, prob):
```

Comment:
Please look at Class Diagram, it is updated now.

There you will see a new `mutate(self, solution, mode)` function. It is basically the same, i.e just wrapper around these three `mutate_1`, `mutate_2`, `mutate_3`.

Also, please try to follow function parameters and return values from the Class Diagram.